products/BellHybrid/alarms/src/actions/PlayAudioActions.cpp

```diff
@@ -24,8 +24,15 @@ namespace alarms
         if (duration) {
             spawnTimer(*duration);
         }
-        return service.bus.sendUnicast(std::make_shared<service::AudioStartPlaybackRequest>(path, playbackType),
-                                       service::audioServiceName);
+        const auto fadeInSettings = utils::getNumericValue<bool>(
+            settings.getValue(bell::settings::Alarm::fadeActive, settings::SettingsScope::Global));
+        const auto fadeInEnabled =
+            fadeInSettings
```

Comment:
This nested ternary condition is IMO quite hard to read, if I understand correctly it should be equivalent to 

```
const auto fadeInEnabled = (fadeInSettings && (playbackType == audio::PlaybackType::Alarm)) ? audio::FadeIn::Enable : audio::FadeIn::Disable;
```

Reply:
done